.travis/linux_tengine_runner.sh

```diff
@@ -259,6 +259,12 @@ do_install() {
 
     sudo luarocks install luacheck > build.log 2>&1 || (cat build.log && exit 1)
 
+    wget https://github.com/etcd-io/etcd/releases/download/v3.4.0/etcd-v3.4.0-linux-amd64.tar.gz
+    tar xf etcd-v3.4.0-linux-amd64.tar.gz
+    sudo cp etcd-v3.4.0-linux-amd64/etcd /usr/local/bin/
+    sudo cp etcd-v3.4.0-linux-amd64/etcdctl /usr/local/bin/
+    rm -rf etcd-v3.4.0-linux-amd64
```

Comment:
repeat many times, can we move them to one script?

Reply:
Duplication is acceptable here so that each use case script is independent.

Reply:
fixed by @nic-chen thx